kitsune/wiki/utils.py

```diff
@@ -103,10 +109,11 @@ def _active_contributors_id(from_date, to_date, locale, product):
     return set(list(editors) + list(reviewers))
 
 
-def get_featured_articles(product=None, locale=settings.WIKI_DEFAULT_LANGUAGE):
+def get_featured_articles(product=None, locale=settings.WIKI_DEFAULT_LANGUAGE, topic=None):
```

Comment:
I would change the signature to  `product, topic, locale`when it comes to the order of args

Reply:
The topic arg can be an iterable. This way you can have a single query and separate the articles in memory, similarly to what you did to the `build_topics_data` function

Reply:
I have made some changes that i think improve things.